services/job_chat/evaluation/test_prompts_issue_97_a.ipynb

```diff
@@ -0,0 +1,966 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(results)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 340,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "V1 success rate: 0.8888888888888888, V2 success rate: 0.9\n"
```

Comment:
The success rate here on v1 is way too high!

I think that means you'll need to manually figure out questions which showcase the issue. Or maybe generate many more questions.

We can only solve the problem if we have a good dataset of questions which showcase the issue :)